docs/01-getting-started.md

```diff
@@ -122,10 +122,11 @@ tail /var/log/cloud-init-output.log
 Once it has finished the EC2 will restart. Once it's restarted you can use the Sleeper CLI. Reconnect to the EC2
 with `sleeper environment connect`.
 
-You can access a built copy of the Sleeper scripts by running `sleeper deployment` in the EC2. That will get you a shell
-inside a Docker container inside the EC2. You can run all the deployment scripts there as explained below. If you run it
-outside of the EC2, you'll get the same thing but in your local Docker host. Use the one in the EC2 to avoid the
-deployment being slow uploading jars and Docker images.
+
+You can access a local copy of the Sleeper Git repository by running `sleeper builder` in the EC2. That will get you a
+shell inside a Docker container inside the EC2, with the dependencies for building Sleeper. You can run all the deployment
+scripts there as explained below. If you run it outside of the EC2, you'll get the same thing but in your local
+Docker host. Use the one in the EC2 to avoid the deployment being slow uploading jars and Docker images.
 
 The Sleeper Git repository will also be cloned, and you can access it by running `sleeper builder` in the EC2.
```

Comment:
I think this might be clearer if it was before the paragraph above. Right now the first sentence of the paragraph above seems a little misleading. It might help to replace the "also" here with an explanation that the repository is cloned during the cloud init.